### PR TITLE
minor change in build.gradle.kts

### PR DESCRIPTION
**Pull request by MasterProvider**

### CODE REVIEW

File: main/build.gradle.kts [hunk 1]
```diff
@@ -6,11 +6,8 @@
...
     id("com.android.application")
     id("checkstyle")
-}
-
-apply {
-    plugin("kotlin-android")
-    plugin("kotlin-android-extensions")
+    kotlin("android")
+    kotlin("android.extensions")
 }
```
Comment (schwabe) [inline main/build.gradle.kts:11]: Could you elaborate a bit more why this change should be done? Is it style, does it fix something, does it improve something?

Comment (MasterProvider) [reply main/build.gradle.kts:11]: To make it convenient syntax. :)

**Comment by schwabe** [comment]
Your commit still lack proper commit messages.

File: main/build.gradle.kts (hunk 1 above)
Comment (MasterProvider) [reply main/build.gradle.kts:11]: To make the command line easier and shorter and add the CheckstyleExtension again so that I know what note to do better

Comment (schwabe) [reply main/build.gradle.kts:11]: The commit messages themselves are just "minor change in build.gradle.kts"
